README.md

```diff
@@ -82,11 +82,25 @@ people shouldn't have indefinite access to, like a share link.
 => nil
 
 >> explicit_expiring_sgid = SecretAgentMessage.find(5).to_sgid(expires_at: Time.now.advance(hours: 1))
-=> #<SignedGlobalID:0x008fde45df8937
+=> #<SignedGlobalID:0x008fde45df8937 ...>
 
 # 1 hour later...
 >> GlobalID::Locator.locate_signed explicit_expiring_sgid
 => nil
+
+>> never_expiring_sgid = Document.find(5).to_sgid(expires_in: nil)
+=> #<SignedGlobalID:0x008fde45df8937 ...>
+
+# Any time later...
+>> GlobalID::Locator.locate_signed never_expiring_sgid
+=> #<Document:0x007fae94bf6298 @id="5">
+```
+
+Alter the default SGID lifetime in app configuration, like so:
+
+```ruby
+# config/initializers/global_id.rb
+Rails.application.config.global_id.expires_in = 3.months
```

Comment:
Fine with adding Rails specific examples, but we should add a pure library version. Ala:

> You can extend or otherwise change the lifetime of an SGID by setting the class level `expires_in`:

```ruby
SignedGlobalID.expires_in = 3.months # Make signed ids live 3 times longer than usual
```

After we can show the Rails example with a preamble:

> If you're using Global ID in a Rails app you could set the value in an initializer:

Then the example.